lib/tasks/url_to_subscriber_list_criteria.rake

```diff
@@ -0,0 +1,74 @@
+desc "Migrate govuk delivery topics to email alert api"
+task url_to_subscriber_list_criteria: :environment do
+  require 'gds_api/helpers'
+  include GdsApi::Helpers
+  require 'pp'
+
+  parsed = 0
+  skipped = Hash.new(0)
+  missing_lookup = 0
+
+  static_data = UrlToSubscriberListCriteria::BulkStaticData.new
+
+  CSV.foreach(ARGV[-1], headers: true) do |row|
+    url = row['_id']
+    if url =~ %r{/government/policies/.*/activity.atom} # these have already been migrated
+      skipped[:policy_activity] += 1
+    elsif url =~ %r{/government/policies.atom} # these have already been migrated
+      skipped[:policy] += 1
+    elsif url =~ /official_document_status=/ # to be removed
+      skipped[:official_document_status] += 1
+    elsif url =~ /relevant_to_local_government=1/ # to be removed
+      skipped[:relevant_to_local_government] += 1
+    elsif url =~ /finder-frontend.production.alphagov.co.uk/
+      skipped[:finder] += 1
+    else
+      parser = UrlToSubscriberListCriteria.new(url, static_data)
+      if ENV['MIGRATE_TO_EMAIL_API']
+        criteria = parser.convert.merge('gov_delivery_id' => row['topic_id'])
+
+        response = email_alert_api.find_or_create_subscriber_list(criteria)
```

Comment:
Missing lookups have been dealt with and will have better reporting - as per below.

We would only need to add `created_at` to `email_alert_api`, `gds-api-adapters` only whitelists the search params..